packages/gasket-plugin-express/lib/index.js

```diff
@@ -78,7 +78,7 @@ module.exports = {
 
       const middlewares = [];
       await gasket.execApply('middleware', async (plugin, handler) => {
-        const middleware = handler(app);
+        const middleware = await handler(app);
```

Comment:
Can you do this same treatment for fastify?
https://github.com/godaddy/gasket/blob/main/packages/gasket-plugin-fastify/lib/index.js#L64

It'd be nice to centralize this somehow eventually.

Reply:
Duplicated fix for fastify